components/Voting/EloVote.styled.tsx

```diff
@@ -5,21 +5,26 @@ import { calcRem } from 'components/utils/styles/calcRem'
 
 export const StyledEloVoteContainer = styled('div')(({ theme }) => ({
   display: 'grid',
-  gridTemplateColumns: `repeat(2, minmax(${calcRem(80)}, 1fr))`,
-  gridTemplateRows: '100%',
+  gridTemplateColumns: `repeat(1, minmax(${calcRem(80)}, 1fr))`,
   gridColumn: '1 / -1',
   gap: theme.metrics.md,
   inlineSize: '100%',
   maxInlineSize: calcRem(965),
-  maxBlockSize: '60vh',
+  /*maxBlockSize: '60vh',*/
   paddingInlineStart: calcRem(theme.metrics.sm),
   paddingInlineEnd: calcRem(theme.metrics.sm),
   marginInlineStart: 'auto',
   marginInlineEnd: 'auto',
+  overflowY: 'auto',
+  maxHeight: '75vh',
```

Comment:
[minor] `maxBlockSize` is the [logical property](https://css-tricks.com/css-logical-properties/) for `max-height`